mne/io/brainvision/brainvision.py

```diff
@@ -365,7 +365,10 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
     info = _empty_info(sfreq)
 
     # check binary format
-    assert cfg.get('Common Infos', 'DataFormat') == 'BINARY'
+    data_format = cfg.get('Common Infos', 'DataFormat')
+    if data_format != 'BINARY':
+        raise ValueError('Only support for data stored in binary format '
+                         'is supported. Your data are in %s.' % data_format)
```

Comment:
You mean something like

'Only data in binary format is supported. Your data are in %s.'

?